jacques/jacques.py

```diff
@@ -1,27 +1,31 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
```

Comment:
I think we should in principle not need this here -- see suggestion below to try to fix?